README.md

```diff
@@ -90,12 +90,74 @@ When you have addressed all technical review and peer review comments, notify th
 
 The developers check that the **Tech review passed** label has been added to the pull request and peer pull request approval provided, then accept it.
 
-## Installing Pantheon
+## Deploying Pantheon
 
-### Prerequisites
+Install the Pantheon codebase, perform some basic initial configuration, and deploy Pantheon with a containerized MongoDB database.
+
+**Prerequisites**
+* Tested on Red Hat Enterprise Linux (RHEL) 8.3 and Fedora 33.
+* openJDK
 * Podman
-* Buildah
-* Java
+
+**Procedure**
+
+1. Clone the git repo:
+
+        $ git clone https://github.com/redhataccess/pantheon.git
+
+2. Set the `PANTHEON_CODEBASE` environment variable to the root directory of the repo:
+
+        $ export PANTHEON_CODEBASE=<path>/pantheon/
+
+3. Set your `JAVA_HOME` variable:
+
+
+        $ vi ~/.bashrc
+        ...
+        # User specific aliases and functions
+        export JAVA_HOME=$(readlink -f /usr/bin/java | sed "s:bin/java::")
```

Comment:
Probably worth reminding people to log out and log in again or source this variable since we're adding it to .bashrc